utils/message-generator/src/main.rs

```diff
@@ -710,111 +733,123 @@ mod test {
         assert!(true)
     }
 
-    //#[tokio::test]
-    //async fn it_initialize_a_pool_and_connect_to_it() {
-    //    //let mut bitcoind = os_command(
-    //    //    "./test/bin/bitcoind",
-    //    //    vec!["--regtest", "--datadir=./test/appdata/bitcoin_data/"],
-    //    //    ExternalCommandConditions::new_with_timer_secs(10)
-    //    //        .continue_if_std_out_have("sv2 thread start")
-    //    //        .fail_if_anything_on_std_err(),
-    //    //)
-    //    //.await;
-    //    //let mut child = os_command(
-    //    //    "./test/bin/bitcoin-cli",
-    //    //    vec![
-    //    //        "--regtest",
-    //    //        "--datadir=./test/appdata/bitcoin_data/",
-    //    //        "generatetoaddress",
-    //    //        "16",
-    //    //        "bcrt1qttuwhmpa7a0ls5kr3ye6pjc24ng685jvdrksxx",
-    //    //    ],
-    //    //    ExternalCommandConditions::None,
-    //    //)
-    //    //.await;
-    //    //child.unwrap().wait().await.unwrap();
-    //    let mut pool = os_command(
-    //        "cargo",
-    //        vec![
-    //            "llvm-cov",
-    //            "--no-report",
-    //            "run",
-    //            "-p",
-    //            "pool_sv2",
-    //            "--",
-    //            "-c",
-    //            "./test/config/pool-config-sri-tp.toml",
-    //        ],
-    //        ExternalCommandConditions::new_with_timer_secs(60)
-    //            .continue_if_std_out_have("Listening for encrypted connection on: 127.0.0.1:34254"),
-    //    )
-    //    .await;
+    #[tokio::test]
+    // for testing purposes, it may be useful to leave the code for running bitcoind
+    async fn it_initialize_a_pool_and_connect_to_it() {
+        //dont clean the following commented code
+        /*
+        let mut bitcoind = os_command(
+            "./test/bin/bitcoind",
+            vec!["--regtest", "--datadir=./test/appdata/bitcoin_data/"],
+            ExternalCommandConditions::new_with_timer_secs(10)
+                .continue_if_std_out_have("sv2 thread start")
+                .fail_if_anything_on_std_err(),
+        )
+        .await;
+        let mut child = os_command(
+            "./test/bin/bitcoin-cli",
+            vec![
+                "--regtest",
+                "--datadir=./test/appdata/bitcoin_data/",
+                "generatetoaddress",
+                "16",
+                "bcrt1qttuwhmpa7a0ls5kr3ye6pjc24ng685jvdrksxx",
+            ],
+            ExternalCommandConditions::None,
+        )
+        .await;
+        child.unwrap().wait().await.unwrap();
+        */
```

Comment:
why are we keeping these comments?

`//dont clean the following commented code` doesn't give any context

Reply:
because can still be useful in future

Reply:
maybe you want to change the:
        //dont clean the following commented code
with: TODO fix the below tests

the first comment don't say anything about why you want to keep the below comment and people get confused.